### PR TITLE
wallet2_api: implement estimateTransactionFee

### PR DESCRIPTION
**Pull request by xiphon**

### CODE REVIEW

File: src/wallet/api/wallet.cpp [hunk 1]
```diff
@@ -1671,6 +1671,26 @@ void WalletImpl::disposeTransaction(PendingTransaction *t)
     delete t;
 }
 
+uint64_t WalletImpl::estimateTransactionFee(const std::vector<std::pair<std::string, uint64_t>> &destinations,
+                                            PendingTransaction::Priority priority) const
+{
+    const size_t pubkey_size = 34;
+    const size_t encrypted_paymentid_size = 10;
+    const size_t extra_size = pubkey_size + encrypted_paymentid_size;
```
Comment (UkoeHB) [inline src/wallet/api/wallet.cpp]: I believe pubkey size is 33 (32 bytes key + 1 byte tag + 0 bytes length), and encrypted payment id size is 11 (8 bytes encrypted ID + 1 byte extra nonce tag + 1 byte extra nonce length + 1 byte payment ID extra nonce tag).

**Comment by Gingeropolous** [comment]
> The wallet2 method could have just called the existing to avoid having to plump up the diff, but it's small enough.

wait, does that mean if that the method has to be changed in the future, it now has to be changed twice?

**Comment by xiphon** [comment]
> it now has to be changed twice?

no

File: src/wallet/api/wallet.cpp (hunk 1 above)
Comment (xiphon) [reply src/wallet/api/wallet.cpp]: Updated

**Pull request merged**